### PR TITLE
Fix rollbar dashboard

### PR DESCRIPTION
**Pull request by ragurney**

### CODE REVIEW

File: test/lib/samson/form_builder_test.rb [hunk 1]
```diff
@@ -145,28 +145,38 @@ def fake_erb_rendering
   end
 
   describe '#fields_for_many' do
-    before { template.expects(:delete_checkbox).returns('delete_checkbox') }
     def render(args)
+      project.rollbar_dashboards_settings.build
       builder.fields_for_many(*args) do |p|
-        p.text_field :name
+        p.text_field :base_url, placeholder: 'thing!'
```
Comment (dragonfax) [inline test/lib/samson/form_builder_test.rb]: thing?

File: lib/samson/form_builder.rb [hunk 2]
```diff
@@ -64,7 +64,7 @@ def fields_for_many(association, description, add_rows_allowed: false)
       content << fields_for(association) do |a|
         content_tag(:div, class: 'form-group') do
           yield(a)
-          @template.delete_checkbox a
+          @template.concat @template.delete_checkbox a
```
Comment (dragonfax) [inline lib/samson/form_builder.rb]: Does this make it a link?

File: test/lib/samson/form_builder_test.rb (hunk 1 above)
Comment (dragonfax) [reply test/lib/samson/form_builder_test.rb]: Sorry, forgot I was reading a test.

File: lib/samson/form_builder.rb (hunk 2 above)
Comment (ragurney) [reply lib/samson/form_builder.rb]: It creates a checkbox input

**Pull request merged**